infra/assembler.go

```diff
@@ -33,15 +37,17 @@ func (a *Assembler) sign(e *Elecments) *Elecments {
 	}
 
 	e.SignedProp = sprop
+
 	return e
 }
 
-func (a *Assembler) StartSigner(raw, signed chan *Elecments, done <-chan struct{}) {
+func (a *Assembler) StartSigner(raw chan *Elecments, signed []chan *Elecments, done <-chan struct{}) {
 	for {
 		select {
 		case r := <-raw:
-			signed <- a.sign(r)
-
+			for _, v := range signed {
+				v <- a.sign(r)
```

Comment:
minor possible improvement - we could sign only once here, and send result multiple times